vm/nova/vm.go

```diff
@@ -447,14 +372,23 @@ func accountBlockIssuerSTVF(vmParams *vm.Params, input *vm.ChainOutputWithIDs, c
 	manaIn := vmParams.WorkingSet.TotalManaIn
 	manaOut := vmParams.WorkingSet.TotalManaOut
 
+	// add the claimed mana rewards of the account in question
+	if manaReward, isClaiming := vmParams.WorkingSet.Rewards[next.ChainID()]; isClaiming {
```

Comment:
@cyberphysic4l @PhilippGackstatter please have a closer look here. I think that was missing before, because we never checked the staking claiming together with the block issuer feature correctly.

Reply:
Is this maybe missing somewhere else as well in the codebase?

Reply:
Good point. This also needs to be included in the TotalManaIn calculation upon initialisation of the working set, and perhaps elsewhere.
I do think that rewards should be locked to the account that has earned them, but that does not necessarily need to be the case.
We also need to include this in the account TIP.

Reply:
Added the rewards to `TotalManaIn` now. 
I propose that we lock rewards to the account that earned them, so we would add keep this block of code that Max added, and we update TIP 42 so that `AccountIn = AccountInPotential + AccountInStored + AccountInRewards`.

Reply:
We decided that we don't want to lock the mana to the account, because it does not contribute to the throughput anyway.